jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -31,18 +34,20 @@ enum PayloadTypeDef {
 
 struct SerdeMemberPublicKey(chain_vote::MemberPublicKey);
 
+pub const MEMBER_PUBLIC_KEY_BECH32_HRP: &str = "p256k1_votepk";
+
 impl<'de> Deserialize<'de> for SerdeMemberPublicKey {
     fn deserialize<D>(deserializer: D) -> Result<Self, <D as Deserializer<'de>>::Error>
     where
         D: Deserializer<'de>,
     {
-        struct PublicKeyVisitor;
-        impl<'de> Visitor<'de> for PublicKeyVisitor {
+        struct Bech32Visitor;
+        impl<'de> Visitor<'de> for Bech32Visitor {
             type Value = SerdeMemberPublicKey;
 
             fn expecting(&self, formatter: &mut std::fmt::Formatter) -> std::fmt::Result {
                 formatter
-                    .write_str("Expected a compatible hex representation of required public key")
+                    .write_str("Expected a compatible bech32 representation of required public key")
```

Comment:
Remove "Expected ", it will result in a bad error message. Serde API wants this string to complete the phrase "Expecting... "